publishable/assets/css/rtl.css

```diff
@@ -325,3 +325,26 @@
         float: right;
     }
 }
+
+
+[dir="rtl"] .app-container .content-container .navbar-top,
+[dir="rtl"] .app-container .side-body,
+[dir="rtl"] .app-container.expanded .content-container .navbar-top {
+    transition: all .5s cubic-bezier(.19,1,.22,1);
```

Comment:
Voyager does not use this transition when opening the user-dropdown, so I think neither should RTL-mode.

Reply:
Sorry for the late response. I will look at it when I finish my work.